corehq/apps/domain/views.py

```diff
@@ -1777,7 +1766,6 @@ def page_context(self):
             'fixture_forms': fixture_forms,
             'fixture_ids': [data.id for data, form in fixture_forms],
             'can_publish_as_org': self.can_publish_as_org,
-            'autocomplete_fields': ('project_type', 'phone_model', 'user_type', 'city', 'countries', 'region'),
```

Comment:
The only one of these fields that's actually part of the [SnapshotSettingsForm](https://github.com/dimagi/commcare-hq/blob/e85086957a7b5c22992ac197058620693fdc5583/corehq/apps/domain/forms.py#L231) is `project_type`, and on prod that isn't even an autocomplete, it just throws a js error.